prijateli_tree/app/routers/games.py

```diff
@@ -400,18 +416,37 @@ def route_add_score(
     """
     Function that updates the player's score in the database
     """
+    game, player = get_game_and_player(game_id, player_id, db)
+    game_status = did_player_win(game, player_id, db)
+    session_player = get_session_player_from_player(player, db)
+
+    session_player.correct_answers += game_status["is_correct"]
+    session_player.points += game_status["is_correct"] * WINNING_SCORE
+    db.commit()
+    db.refresh(session_player)
 
-    # There isn't a place to store this right now as far as I can tell
-    # player_session = ...
+    redirect_url = URL("games/{game_id}/player/{player_id}/end_of_game")
 
-    # result = did_player_win(game_id, player_id, db)
+    return RedirectResponse(url=redirect_url, status_code=HTTPStatus.FOUND)
 
-    # we want to count the number of games they are correct, e.g.
-    # player_session.n_correct += result["is_correct"]
-    # player_session.total_points += result["is_correct"] * WINNING_SCORE
-    url = "/{game_id}/player/{player_id}/end_of_game"
 
-    return RedirectResponse(url=url, status_code=HTTPStatus.FOUND)
+@router.get("/current_score/{session_player_id}")
+def route_get_score(
```

Comment:
I'd try to be mindful of which player ID you're using as it can make development tricky if you're constantly switching between the two. The `GamePlayer` id can always get you the `session_player_id`, so I'd go that way if it were up to me.